lib/hexpm_web/controllers/api/organization_controller.ex

```diff
@@ -50,6 +50,14 @@ defmodule HexpmWeb.API.OrganizationController do
     end
   end
 
+  def audit_logs(conn, params) do
+    organization = conn.assigns.organization
+    audit_logs = AuditLogs.all_by(organization, Hexpm.Utils.safe_int(params["page"]))
+
+    conn
+    |> render(:audit_logs, audit_logs: audit_logs)
```

Comment:
No need to pipe in the case where it's a single pipe. We find no pipe more readable in those cases.

Reply:
Fixed in dec9aa42 Do not use pipe for a single pipe in OrganizationController#audit_logs